packages/docusaurus-theme-classic/src/theme/NavbarItem/LocaleDropdownNavbarItem/index.tsx

```diff
@@ -48,7 +49,13 @@ export default function LocaleDropdownNavbarItem({
   const items = [...dropdownItemsBefore, ...localeItems, ...dropdownItemsAfter];
 
   // Mobile is handled a bit differently
-  const dropdownLabel = mobile ? 'Languages' : getLocaleLabel(currentLocale);
+  const dropdownLabel = mobile
+    ? translate({
+        message: 'Languages',
+        id: 'theme.navbar.mobileLanguageDropdown.label',
```

Comment:
I am wondering if we shouldn't add this to something like "theme.common.languages" or something, as we'll likely add language switchers in other places (footer for example) later?

We probably don't want to duplicate the label and have to retranslate or rename the key later? (it is a breaking change)

maybe I'm anticipating too much and we'll see later

Reply:
Let's see this later... Remember that we never have footer dropdowns in the first place, so we are not even sure what its UX is like. Maybe it will be more like the desktop navbar item where the label is the current language's label